testing/sdk_marathon.py

```diff
@@ -52,19 +53,29 @@ def get_config(app_name, timeout=TIMEOUT_SECONDS):
     return config
 
 
-class _deployment_result(object):
-    def __init__(self, version, errmsg):
-        self._version = version
-        self._errmsg = errmsg
+class MarathonDeploymentResponse:
+    def __init__(self, response: requests.Response) -> None:
+        try:
+            response_json = response.json()
+        except ValueError:
+            log.error("Failed to parse marathon response as JSON: %s", response.text)
+            raise
+        log.info("Marathon deployment response JSON: %s", response_json)
+        response.raise_for_status()
+
+        self._version = response_json["version"]
+        if response.status_code == 201:
+            self.deployment_id = self._parse_deployment_id_on_app_creation(response_json)
```

Comment:
There's a mismatch in the member names here. This should probably be `_deployment_id`.